pydoctor/test/test_commandline.py

```diff
@@ -205,3 +221,26 @@ def test_main_source_outside_basedir(capsys: CapSys) -> None:
             'pydoctor/test/testpackages/basic/'
             ])
     assert "Source path lies outside base directory:" in capsys.readouterr().err
+
+
+def test_make_intersphix(tmp_path: Path) -> None:
+    """
+    --make-interphinx will only produce the Sphinx inventory object.
+
+    This is also an integration test for Sphinx inventory writer.
+    """
+    inventory = tmp_path / 'objects.inv'
+    exit_code = driver.main(args=[
+        '--project-base-dir=.',
+        '--make-intersphinx',
+        '--project-name=acme-lib',
+        '--project-version=1.2.0',
```

Comment:
Since the system test already checks using "1.2.0", maybe use something more complex here, like "20.12.0-dev123". Of course that doesn't actually make a difference with the current implementation, since we just pass the string around, but if we ever switch back to parsing versions, it could be useful.